mixins/async-container/test/async-container-mixin.visual-diff.js

```diff
@@ -7,6 +7,16 @@ describe('d2l-async-container-mixin', function() {
 
 	let browser, page;
 
+	const getAsyncStateEvent = (page, selector, state) => {
+		return page.$eval(selector, (elem, state) => {
+			return new Promise((resolve) => {
+				elem.addEventListener('d2l-async-demo-container-changed', (e) => {
```

Comment:
A slightly different flavor of oneEvent could handle this... except instead of checking an animation event property, it's checking some custom data.  🤔